tutorials/hyperspectral/Work-With-Hyperspectral-Data-In-R.Rmd

```diff
@@ -436,153 +438,94 @@ as other GIS and imaging processing software like QGIS and ENVI do.
-
-<br>
-`"UTM,1.000,1.000,256521.000,4112571.000,1.000000e+000,`
-`1.000000e+000,11,North,WGS-84,units=Meters" `. 
-</br>
-
-Notice that this information is separated by commas. We can use the 
-`strsplit` command in R to extract each element into a vector. The elements are 
-position 4 and 5 represent the lower left hand corner of the raster. We need 
-this information to define the raster's extent.
-
-```{r read-map-info}
-
-# Populate the raster image extent value. 
-# get the map info, split out elements
-mapInfo<-h5read(f,"map info")
-
-# Extract each element of the map info information 
-# so we can extract the lower left hand corner coordinates.
-mapInfo<-unlist(strsplit(mapInfo, ","))
-
-# view the attributes in the map dataset
-mapInfo
-
-```
-
-Next we define the extents of our raster. The extents will be used to calculate 
-the raster's resolution. The lower left hand corner is located at mapInfo[4:5]. 
-We can define the final raster dataset extent by adding the number of rows to 
-the Y lower left hand corner coordinate and the number of columns in the 
-`Reflectance` dataset to the X lower left hand corner coordinate. 
+* The spatial extent of the image 
 
-<figure>
-		<a href="{{ site.baseurl }}/images/hyperspectral/sat_image_lat_lon.png">
-		<img src="{{ site.baseurl }}/images/hyperspectral/sat_image_lat_lon.png"></a>
-		<figcaption>The extent of a raster represents the spatial location of each 
-		corner. The coordinate units will be determined by the spatial projection/
-		coordinate reference system that the data are in. Source: National Ecological
-		Observatory Network (NEON) </figcaption>
-</figure>
-
-<a href="{{ site.baseurl }}/raster-data-series" target="_blank"> Learn more about raster attributes including extent, and coordinate reference systems here.</a>
 
 ### Define Raster CRS
 
-We have defined the extent of our raster but we still need to define the Coordinate 
-reference system (`CRS`) of the raster. To do that, we can first grab the CRS
-string from the HDF5 attributes. Then we can assign that CRS to the raster object.
+First, we need to define the Coordinate reference system (`CRS`) of the raster. 
+To do that, we can first grab the EPSG code from the HDF5 attributes, and covert the
+EPSG to a CRS string. Then we can assign that CRS to the raster object.
 
 ```{r define-CRS}
 
-# Create the projection in as object
-myCRS <- spInfo$projdef
-myCRS
+# Extract the EPSG from the h5 dataset
+myEPSG=h5read(f, "/SJER/Reflectance/Metadata/Coordinate_System/EPSG Code")
+
+# convert the EPSG code to a CRS string
+myCRS=crs(paste0("+init=epsg:",myEPSG))
 
 # define final raster with projection info 
 # note that capitalization will throw errors on a MAC.
 # if UTM is all caps it might cause an error!
-b34r <- raster(b34, 
+b9r <- raster(b9, 
         crs=myCRS)
+b9r
 
-b34r
-
-#let's have a look at our properly positioned raster. Take note of the 
+#let's have a look at our properly oriented raster. Take note of the 
```

Comment:
space between # and comment, please.  (Correct throughout for others I've missed).

Reply:
good catch! Corrected here and a few other places throughout